scripts/sscm.py

```diff
@@ -0,0 +1,773 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import os
+import random
+import re
+import shutil
+import subprocess
+import sys
+import textwrap
+import time
+import traceback
+
+
+"""
+Simple Scylla Cluster Manager
+
+See ./sscm.py --help
+
+For usage from python, see SupervisorClient.
+"""
+
+
+LOG_FILE = "supervisor.log"
+PID_FILE = "supervisor.pid"
+CONFIG_FILE = "config.json"
+FIFO_IN = "supervisor_in"
+FIFO_OUT = "supervisor_out"
+
+SCYLLA_LOG_FILE = "system.log"
+
+
+def _parse_args(app_name, argv):
+    app_name = os.path.basename(app_name)
+
+    parser = argparse.ArgumentParser(
+            prog=f"{app_name}",
+            formatter_class=argparse.RawDescriptionHelpFormatter,
+            description=textwrap.dedent(f"""\
+            Simple Scylla Cluster Manager
```

Comment:
The text blob should be indented more, away from the Python statements

Reply:
Done in v2.